Add visiting page (#289)
ournetworks/ournetworks.ca: visiting.md

---
layout: page
order: 2
title: "Visiting Vancouver"
titleDisplay: "Visiting"
---

- [Getting around](#getting-around)
  - [Cycling](#cycling)
  - [Public transit](#public-transit)
- [Staying in Vancouver](#staying-in-vancouver)
  - [Unionized hotels](#unionized-hotels)
  - [Small-business or IBPOC-owned hotels](#small-business-or-ibpoc-owned-hotels)
  - [Mitigate the harm of short-term rentals](#mitigate-the-harm-of-short-term-rentals)

---

Our event is taking place in the Downtown Eastside (DTES), steps from the heart of Chinatown in the City of Vancouver. DTES is one of the oldest neighbourhoods, including an original civic centre right around our event venue, with the original industrial area further east with the port along the waterfront. The City [has further details and context](https://vancouver.ca/news-calendar/downtown-eastside.aspx) about the neighbourhood today:

> > In recent years, the Downtown Eastside has struggled with many complex challenges including drug use, crime, homelessness, housing issues, unemployment, and loss of businesses in the community.

Reviewer: styling may need to be adjusted here

{:.font-display}

## Getting around

Vancouver is easy to travel around by public transit, bicycle, and on foot around the event venue.

{:.font-display}

### Cycling

Cycling even as a visitor is possible with [Mobi](https://www.mobibikes.ca/), a city-wide bike share program owned by Rogers, which has an app and a system map with bike stations throughout the downtown area. Both are membership and pass based. There are monthly passes for $49 which include unlimited 30-minute rides or, or pay-per-ride pricing that starts at $3.00 and is priced per minute. Otherwise, plenty of local bike shops also offer week long rentals.

> _Please note, wearing a helmet while bicycling is the law in British Columbia and you can be fined ($29) for failing to do so. You can learn more about cycling in the province from [HUB Cycling](https://bikehub.ca/sites/default/files/imce/hub_cyclist_handbook_web.pdf)._

Reviewer: also may want to chantge this.

{:.font-display}
### Public transit

> Public transpit is operated by [Translink](https://www.translink.ca/) and the network consists of automated rapid transit called the Skytrain, Seabus ferries, commuter rail, and bus. Translink offers a [trip planner](https://www.translink.ca/trip-planner) and [route maps](https://www.translink.ca/schedules-and-maps) to aide passengers, and routes are searchable from most transit apps. 

Reviewer: ```suggestion
Public transit is operated by [Translink](https://www.translink.ca/) and the network includes automated rapid transit called the Skytrain, Seabus ferries, commuter rail, and bus. Translink has a [trip planner](https://www.translink.ca/trip-planner) and [route maps](https://www.translink.ca/schedules-and-maps) to aide passengers, and routes are searchable from most transit apps. 
```

> **Fares** are for the whole system and allow transfers between bus, Skytrain, and more on a single trip within 90 minutes. You can pay by cash, tap, and [Compass Card](https://www.compasscard.ca/). There are three [fare zones](https://www.translink.ca/transit-fares/pricing-and-fare-zones), with a single 1-ZONE trip costing $3.15 for cash or tap and $2.55 if you use the Compass card. 

Reviewer: ```suggestion
**Fares** are for the whole system and allow transfers on 90 minute trips. You can pay by cash, tap, and load funds on a [Compass Card](https://www.compasscard.ca/). There are three [fare zones](https://www.translink.ca/transit-fares/pricing-and-fare-zones), with a single 1-ZONE trip costing $3.15 by cash or tap and $2.55 by Compass. 
```

> The airport is [Vancouver International airport (YVR)](https://www.yvr.ca/), from which you can travel to downtown Vancouver or across the Metro area via Skytrain along the [Canada Line](https://www.translink.ca/schedules-and-maps/skytrain). There is an [additional $5 fare](https://www.translink.ca/transit-fares/transferring-and-addfare) for taking the Skytrain from the airport.

Reviewer: ```suggestion
The airport is [Vancouver International airport (YVR)](https://www.yvr.ca/), you can travel to downtown Vancouver or across the Metro area from it via Skytrain on the [Canada Line](https://www.translink.ca/schedules-and-maps/skytrain). There is an [additional $5 fare](https://www.translink.ca/transit-fares/transferring-and-addfare) for taking the Skytrain from the airport.
```

{:.font-display}

## Staying in Vancouver

> For those looking at alternative accommodations including platforms like Airbnb, we wanted to provide context about the current housing market. 

Reviewer: ```suggestion
For those looking at accommodations, including short-term rentals on platforms like Airbnb, we wanted to provide context about the current housing market. 
```

> Vancouver is an expensive place to live and visit. Like many cities around the world, Vancouver is experiencing an [affordability crisis](https://globalnews.ca/news/10401449/vancouver-full-blown-crisis-housing-affordability-report/) compounded by a decades-long [housing crisis](https://thetyee.ca/Culture/2023/05/03/What-Behind-Housing-Crisis-Canada/), in part driven by housing that would otherwise be available to rentals being treated as speculative investments for the operation of [short-term rentals](https://www.cbc.ca/news/canada/british-columbia/short-term-rental-accomodation-act-bc-rules-and-enforcement-1.7177836). As a result, Vancouver rent is the [most expensive in Canada](https://vancouversun.com/news/local-news/vancouver-rent-flat-decline-most-expensive-report), an increase in [bad-faith evictions](https://www.theglobeandmail.com/real-estate/vancouver/article-landlord-use-now-most-frequently-used-tool-to-evict-bc-renters/), and [tenant unions](https://www.vancouvertenantsunion.ca/).

Reviewer: ```suggestion
Vancouver is an expensive place to live and visit. Like many cities around the world, Vancouver is experiencing an [affordability crisis](https://globalnews.ca/news/10401449/vancouver-full-blown-crisis-housing-affordability-report/) compounded by a decades-long [housing crisis](https://thetyee.ca/Culture/2023/05/03/What-Behind-Housing-Crisis-Canada/), in part driven by housing that would otherwise be available to rentals being treated as speculative investments for the operation of [short-term rentals](https://www.cbc.ca/news/canada/british-columbia/short-term-rental-accomodation-act-bc-rules-and-enforcement-1.7177836). As a result, Vancouver rent is the [most expensive in Canada](https://vancouversun.com/news/local-news/vancouver-rent-flat-decline-most-expensive-report) and there has been an increase in [bad-faith evictions](https://www.theglobeandmail.com/real-estate/vancouver/article-landlord-use-now-most-frequently-used-tool-to-evict-bc-renters/). The [Vancouver Tenants Union](https://www.vancouvertenantsunion.ca/about) has members who live in the 50% of the city that is rentals and collectively ensures their rights as tenants are respected.
```

> As of May 1, 2024, B.C. has introduced new rules that [apply to Vancouver](https://vancouver.ca/doing-business/short-term-rentals.aspx): short-term rentals are restricted to the home the operator lives in, and they must be rented out for less than 30 consecutive days at a time. s

Reviewer: ```suggestion
As of May 1, 2024, B.C. has new rules that [apply to Vancouver](https://vancouver.ca/doing-business/short-term-rentals.aspx): short-term rentals are restricted to the home the operator lives in and they must be rented out for less than 30 consecutive days at a time. 
```

Inspired by [4S 2019 in New Orleans](https://www.4s2019.org/accommodation/), we have [once again](https://2019.ournetworks.ca/visiting/) provided information on ways to mitigate the impact of short-term rentals as well as notes on how to make your own guides.

{:.font-display}

### Unionized hotels

> We checked [Fair Hotels](https://www.fairhotel.org/search?location%5Bgeolocation_geocoder_address%5D=Vancouver%2C+BC%2C+Canada&location%5Blat_north_east%5D=49.3172939&location%5Blng_north_east%5D=-123.023068&location%5Blat_south_west%5D=49.19817700000001&location%5Blng_south_west%5D=-123.22474&brand=All&field_guest_rooms_value%5Bmin%5D=0&field_guest_rooms_value%5Bmax%5D=6000&meeting_rooms%5Bmin%5D=0&meeting_rooms%5Bmax%5D=500&venue_size%5Bmin%5D=0&venue_size%5Bmax%5D=3000000), and as of May 2024, these hotels are unionized and less than $500/night,roughly sorted by lowest price and distance:

Reviewer: ```suggestion
We checked [Fair Hotels](https://www.fairhotel.org/search?location%5Bgeolocation_geocoder_address%5D=Vancouver%2C+BC%2C+Canada&location%5Blat_north_east%5D=49.3172939&location%5Blng_north_east%5D=-123.023068&location%5Blat_south_west%5D=49.19817700000001&location%5Blng_south_west%5D=-123.22474&brand=All&field_guest_rooms_value%5Bmin%5D=0&field_guest_rooms_value%5Bmax%5D=6000&meeting_rooms%5Bmin%5D=0&meeting_rooms%5Bmax%5D=500&venue_size%5Bmin%5D=0&venue_size%5Bmax%5D=3000000), and these hotels are unionized and less than $500/night,roughly sorted by lowest price and distance:
```

- [St. Regis Hotel](https://www.stregishotel.com)
- [Holiday Inn Vancouver-Centre (Broadway)](https://www.ihg.com/holidayinn/hotels/us/en/vancouver/yvrbw/hoteldetail)
- [Hilton Vancouver Metrotown](https://www.hilton.com/en/hotels/yvrvmhf-hilton-vancouver-metrotown/) (along the SkyTrain line)
- [Inn At the Quay](https://bookings.innatwestminsterquay.com/) (along the SkyTrain line)

> In addition, [Unite Here Local 40](https://www.uniteherelocal40.org/) has asked that people **do not** cross their picketline at the following locations:

Reviewer: ```suggestion
In addition, [Unite Here Local 40](https://www.uniteherelocal40.org/) has asked that people **do not** cross their picket lines at the following locations:
```

- Sheraton YVR Hotel
- Marriott YVR Hotel
- Hilton YVR Hotel
- Radisson Blu YVR Hotel

{:.font-display}

### Small-business or IBPOC-owned hotels

We searched and could not confirm a directory of accommodations that are small businesses or IBPOC-owned in Vancouver. If you know of any, please contact us at [orga@ournetworks.ca](mailto:orga@ournetworks.ca).

{:.font-display}

### Mitigate the harm of short-term rentals

> Considerations for other short-term rentals:

Reviewer: ```suggestion
Considerations for short-term rentals:
```
